inky-build-pkg/action.yaml

```diff
@@ -52,6 +43,6 @@ runs:
         repository-append: ${{ github.workspace }}/packages
         keyring-append: /etc/apk/keys/${{ inputs.signing-key-name }}.pub
         workspace-dir: ${{ github.workspace }}/build/${{ inputs.package-name }}
-        empty-workspace: ${{ inputs.empty-workspace }}
         signing-key-path: ${{ github.workspace }}/${{ inputs.signing-key-name }}
-        source-dir: ${{ inputs.source-dir }}
+        source-dir: ${{ github.workspace }}/${{ inputs.package-name }}
+        empty-workspace: false
```

Comment:
The one thing that threw me reading this and the actions.yaml (we might want to consider adding a README there?) that we use was that we create this workspace now always on L36 (so I'd think it's empty?) and we don't touch anything there (or I missed it somehow), so was curious why this isn't true, and that I think could use more documentation so I could understand what empty-workspace means. Consider this just a drive-by commentary since I saw it on slack, and I'm not very familiar with this at all :)